aten/src/ATen/native/sparse/SparseBinaryOpIntersectionCommon.h

```diff
@@ -275,8 +275,11 @@ void _sparse_binary_op_intersection_kernel_impl(
       KernelLauncher::launch(iter,
           // NOTE: capture by value required by CUDA
           [=] FUNCAPI (index_t nnz_idx) -> int64_t {
-          const auto* RESTRICT ptr_indices_dim = ptr_indices ? ptr_indices + nnz_idx * indices_nnz_stride : nullptr;
           int64_t hash = 0;
+          if (!ptr_indices) {
+            return hash;
+          }
+          const auto* RESTRICT ptr_indices_dim = ptr_indices + nnz_idx * indices_nnz_stride;
```

Comment:
Alternatively, this can be checked on the tensor level without launching any kernels. Then we can assume that we are not dealing with `nullptr`.